src/components/SingleArticle.jsx

```diff
@@ -36,6 +39,17 @@ const SingleArticle = ({ setVotes }) => {
     }
   };
 
+  const handleAddCommentClick = (e) => {
+    addComment(article_id, { username, comment: commentBody })
+      .then((response) => {
+        setUsername("");
+        setCommentBody("");
+      })
+      .catch((error) => {
+        console.error("Error adding comment:", error);
+      });
```

Comment:
This catch will never be reached, because you've put one in your addComment function. Since all it is doing is `console.log`ging, I'd argue we do not need it.